packages/runtime-core/src/vnode.ts

```diff
@@ -835,6 +835,8 @@ export function mergeProps(...args: (Data & VNodeProps)[]) {
           ret[key] = existing
             ? [].concat(existing as any, incoming as any)
             : incoming
+        } else if (incoming === '') {
+          ret[key] = toMerge[key]
```

Comment:
Maybe more natural to use `incoming` instead of `toMerge[key]` here?
```suggestion
          ret[key] = incoming
```